linkerd/core/src/main/scala/io/buoyant/linkerd/RouterContextFormatter.scala

```diff
@@ -11,18 +11,8 @@ import io.buoyant.router.RoutingFactory.BaseDtab
 
 object RouterContextFormatter {
```

Comment:
Unused import "import io.buoyant.namer.DelegateTree._". It might have been left over from the initial commit of this code.

Reply:
good catch